docs/repositories-configure/codacy-configuration-file.md

```diff
@@ -92,6 +94,30 @@ exclude_paths:
   - "**/*.resource"
 ```
 
+## Including specific files using a Codacy configuration file {: id="include-files"}
+
+The Codacy configuration file allows you to explicitly specify files or directories to include in the analysis. This is particularly useful for [bypassing files or directories that are ignored by default](./ignoring-files.md#default-ignored-files) or specified in `exclude_paths`.
+
+!!! note
+    If both `exclude_paths` and `include_paths` are defined, `include_paths` will specify exceptions to the exclusions defined in `exclude_paths`.
```

Comment:
@codacy-vrhpires Can you confirm this is the case? Thanks

Reply:
@nicklem It is.